ksqldb-engine/src/main/java/io/confluent/ksql/function/udaf/offset/KudafByOffsetUtils.java

```diff
@@ -64,6 +64,29 @@
     }
   };
 
+  static final Comparator<Struct> INTERMEDIATE_STRUCT_COMPARATOR_IGNORE_NULLS =
+      (struct1, struct2) -> {
+        // Ignore nulls: If one of the structs has a null value, then return the other irrespective
+        // of sequence.
```

Comment:
nit: what if they're both null? then we should probably return 0 (I don't think it'll make a difference, but better to be correct)